crates/egui_extras/src/table.rs

```diff
@@ -1010,14 +1010,14 @@ impl<'a> Drop for TableBody<'a> {
 /// The row of a table.
 /// Is created by [`TableRow`] for each created [`TableBody::row`] or each visible row in rows created by calling [`TableBody::rows`].
 pub struct TableRow<'a, 'b> {
-    layout: &'b mut StripLayout<'a>,
+    pub layout: &'b mut StripLayout<'a>,
```

Comment:
What would it mean to change layout in the middle of a row? Should we really support that? Wouldn't a non-mutable accessor be better?